client/jetpack-cloud/sections/partner-portal/license-multi-product-card/index.tsx

```diff
@@ -0,0 +1,195 @@
+	) => void | null;
+	suggestedProduct?: string | null;
+	hideDiscount?: boolean;
+}
+
+export default function LicenseMultiProductCard( props: Props ) {
+	const {
+		tabIndex,
+		products,
+		isSelected,
+		isDisabled,
+		onSelectProduct,
+		suggestedProduct,
+		hideDiscount,
+	} = props;
+	const [ product, setProduct ] = useState( products[ 0 ] );
+	const { setParams, resetParams, getParamValue } = useURLQueryParams();
+	const modalParamValue = getParamValue( LICENSE_INFO_MODAL_ID );
+	const productTitle = getProductTitle( product.name, true );
+	const [ showLightbox, setShowLightbox ] = useState( modalParamValue === product.slug );
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const onSelect = useCallback( () => {
+		if ( isDisabled ) {
+			return;
+		}
+
+		onSelectProduct?.( product );
+	}, [ isDisabled, onSelectProduct, product ] );
+
+	const onKeyDown = useCallback(
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		( e: any ) => {
+			// Spacebar
+			if ( 32 === e.keyCode ) {
+				onSelect();
+			}
+		},
+		[ onSelect ]
+	);
+
+	useEffect( () => {
+		if ( suggestedProduct ) {
+			// Transform the comma-separated list of products to array.
+			const suggestedProducts = suggestedProduct.split( ',' );
+
+			if ( suggestedProducts.includes( product.slug ) ) {
+				onSelect();
+			}
+		}
+	}, [ onSelect, product.slug, suggestedProduct ] );
+
+	const { description: productDescription } = useProductDescription( product.slug );
+
+	const onShowLightbox = useCallback(
+		( e: React.MouseEvent< HTMLElement > ) => {
+			e.stopPropagation();
+
+			dispatch(
+				recordTracksEvent( 'calypso_partner_portal_issue_license_product_view', {
+					product: product.slug,
+				} )
+			);
+
+			setParams( [
+				{
+					key: LICENSE_INFO_MODAL_ID,
+					value: product.slug,
+				},
+			] );
+			setShowLightbox( true );
+		},
+		[ dispatch, product.slug, setParams ]
+	);
+
+	const onHideLightbox = useCallback( () => {
+		resetParams( [ LICENSE_INFO_MODAL_ID ] );
+		setShowLightbox( false );
+	}, [ resetParams ] );
+
+	const variantOptions = products.map( ( option ) => ( {
+		id: option.slug,
+		answerText: getProductVariantShortTitle( option.name ),
+	} ) );
+
+	const onChangeOption = useCallback(
+		( selectedProductSlug: string ) => {
+			if ( isDisabled ) {
+				return;
+			}
+
+			const selectedProduct =
+				products.find( ( { slug } ) => slug === selectedProductSlug ) ?? products[ 0 ];
+
+			if ( isSelected ) {
+				// If the current card is selected, we need to update selected licenses.
+				onSelectProduct?.( selectedProduct, product );
+			}
+
+			setProduct( selectedProduct );
+		},
+		[ isDisabled, isSelected, onSelectProduct, product, products ]
+	);
+
+	return (
+		<>
+			<div
+				className={ classNames( 'license-product-card', 'license-product-card--with-variant', {
+					selected: isSelected,
+					disabled: isDisabled,
+				} ) }
+			>
+				<div className="license-product-card__inner">
```

Comment:
Can we please rename `license-product-card__*` to `license-multi-product-card__*` to match the component name?